bamtk.c

```diff
@@ -72,6 +72,7 @@ int main_import(int argc, char *argv[]);
 int main_samples(int argc, char *argv[]);
 int main_consensus(int argc, char *argv[]);
 int main_reference(int argc, char *argv[]);
+int main_reset(int argc, char *argv[]);     //reset facility implementation
```

Comment:
Can drop the comment as it's out of place compared to all others.

Reply:
Done.